rotkehlchen/db/upgrades/v41_v42.py

```diff
@@ -107,6 +107,17 @@ def _upgrade_evmchains_to_skip_detection(write_cursor: 'DBCursor') -> None:
     )
 
 
+@enter_exit_debug_log()
+def _remove_yearn_events_table(write_cursor: 'DBCursor') -> None:
+    """Delete the table with balancer events"""
+    write_cursor.execute('DROP TABLE yearn_vaults_events')
+    for prefix in ('yearn_vaults_events', 'yearn_vaults_v2_events'):
```

Comment:
Do we have more used_query ranges that start with `yearn_vaults_` ? If not then this can be simplified into one sqlite query right?